account.go

```diff
@@ -929,7 +941,7 @@ func CreateCustomAssetWithKPs(source SeedStr, issuerPair, distPair *keypair.Full
 	if err != nil {
 		return "", "", err
 	}
-	_, _, _, err = createAccountXLM(source, issuerAddr, "5", "")
+	_, _, _, err = SendXLM(source, issuerAddr, "5", "")
```

Comment:
why did these change?

Reply:
createAccountXLM fails if the issuer address already exists. SendXLM handles the case where it might. not a huge deal, just thought it was an easy little tweak to improve idempotence. is there some reason we would want the issuer address not to exist previous to creating a custom asset?

Reply:
@patrickxb ping